core/backend/src/CheckpointManager.ts

```diff
@@ -168,7 +168,7 @@ export class V2CheckpointManager {
         }
       }
 
-      this._cloudCache = CloudSqlite.CloudCaches.getCache({ cacheName: this.cloudCacheName, cacheDir });
+      this._cloudCache = CloudSqlite.CloudCaches.getCache({ cacheName: this.cloudCacheName, cacheDir, cacheSize: "50G" });
```

Comment:
50 what?

Reply:
50 gigabytes, should I add that as a comment?

Reply:
why is it not `50GB` then?
a comment would help

Reply:
It's documented on `CreateCloudCacheArg.cacheSize`.

```
    /** A string that specifies the maximum size of the CloudCache. It should be a number followed by "K",
     * "M" "G", or "T". Default is "10G". */
    cacheSize?: string;
```

Reply:
I don't see a reason to have to document it everywhere it's used too.